class.jetpack.php

```diff
@@ -5312,6 +5312,9 @@ function verify_xml_rpc_signature() {
 		// Let's see if this is onboarding. In such case, use user token type and the provided user id.
 		if ( isset( $this->HTTP_RAW_POST_DATA ) ) {
 			$jpo = json_decode( $this->HTTP_RAW_POST_DATA );
+		} else {
+			$jpo = json_decode( json_encode( $_GET ), FALSE );
```

Comment:
Let's use the standard WP function `wp_json_encode`
https://developer.wordpress.org/reference/functions/wp_json_encode/